internal/configuration/validator/configuration.go

```diff
@@ -3,13 +3,15 @@ package validator
 import (
 	"fmt"
 	"net/url"
+	"os"
 
 	"github.com/authelia/authelia/internal/configuration/schema"
 )
 
 var defaultPort = 8080
 var defaultLogLevel = "info"
 
+//nolint:gocyclo
 // ValidateConfiguration and adapt the configuration read from file.
```

Comment:
```suggestion
// ValidateConfiguration and adapt the configuration read from file.
//nolint:gocyclo // This function is likely to always have lots of if/else statements, as long as we keep the flow clean it should be understandable.
```

Looks like over the last few PR's this is either being dropped or added back in, let's keep what we had originally in there and ensure nolint's are specified after the GoDoc.